src/storage/src/encoding/row_serializer.rs

```diff
@@ -20,12 +20,16 @@ use risingwave_common::types::VirtualNode;
 pub type KeyBytes = Vec<u8>;
 pub type ValueBytes = Vec<u8>;
 
-pub trait RowSerializer {
+pub trait RowEncoding {
```

Comment:
Let's move this trait to `mod.rs`? BTW, `RowEncoding` sounds like a specific encoding approach against `CellBasedEncoding` 🤣. How about just naming it as `Encoding`?